ZhuoHeiChaUI/Events/NotifyAceGoPublicEventArgs.cs

```diff
@@ -7,5 +7,6 @@ namespace ZhuoHeiChaUI.Events
 {
     public class NotifyAceGoPublicEventArgs : EventArgs
     {
+        public bool isPublicAce;
```

Comment:
Same here. Let's use public properties. (And use `PascalCase` too!)

Reply:
Done